ydb/_grpc/grpcwrapper/common_utils.py

```diff
@@ -112,15 +113,16 @@ def __next__(self):
 
 
 class SyncIteratorToAsyncIterator:
```

Comment:
Maybe better `SyncToAsyncIterator`? double `Iterator` looks redundant for me

Reply:
yes, I gree. In outer PR (for split refactoring from bugfix/feature commit)